remotespark/RemoteSparkMagics.py

```diff
@@ -161,14 +160,12 @@ def _get_client_keys(self):
         return "Possible endpoints are: {}".format(self.client_manager.get_endpoints_list())
 
     def _send_command(self, client, command, language):
-        if language == Constants.lang_scala:
-            return client.execute_scala(command)
-        elif language == Constants.lang_pyspark:
-            return client.execute_pyspark(command)
-        elif language == Constants.lang_sql:
-            return client.execute_scala_sql(command)
-        elif language == Constants.lang_pysql:
-            return client.execute_pyspark_sql(command)
+        if language == "sql":
```

Comment:
It's not consistently done that way though. See lines 64-67. If you do not specify a language when using %%sparkmagic, it will take the language of the client/session, which will either be scala or python. This causes the error.  Even if this was done consistently, it's very confusing to have two different meanings for "language" meant to be used at two different places.